magentoerpconnect_order_comment/sale.py

```diff
@@ -96,66 +91,47 @@ class magento_sale_comment(orm.Model):
                    "sale comment on Magento. \nPlease refer to the Magento " \
                    "documentation for details. "
 
-    def _get_comments_from_order(self, cr, uid, ids, context=None):
-        return self.pool['magento.sale.comment'].search(
-            cr, uid, [('magento_sale_order_id', 'in', ids)], context=context)
-
-    _columns = {
-        'openerp_id': fields.many2one(
-            'mail.message',
-            string='Sale Comment',
-            required=True,
-            ondelete='cascade'),
-        'magento_sale_order_id': fields.many2one(
-            'magento.sale.order',
-            'Magento Sale Order',
-            required=True,
-            ondelete='cascade',
-            select=True),
-        'is_customer_notified': fields.boolean(
-            'Customer notified',
-            help=MAGENTO_HELP),
-        'is_visible_on_front': fields.boolean(
-            'Visible on front',
-            help=MAGENTO_HELP),
-        'status': fields.char(
-            'Order status',
-            size=64,
-            help=MAGENTO_HELP),
-        'backend_id': fields.related(
-            'magento_sale_order_id', 'backend_id',
-            type='many2one',
-            relation='magento.backend',
-            string='Magento Backend',
-            store={
-                'magento.sale.comment': (
-                    lambda self, cr, uid, ids, c=None:
-                    ids,
-                    ['magento_sale_order_id'],
-                    10),
-                'magento.sale.order': (
-                    _get_comments_from_order,
-                    ['backend_id'],
-                    20),
-                },
-            readonly=True),
-        'storeid': fields.char(
-            'Store id',
-            help=MAGENTO_HELP),
-    }
-
-    def create(self, cr, uid, vals, context=None):
+    openerp_id = fields.Many2one(
+        'mail.message',
+        string='Sale Comment',
+        required=True,
+        ondelete='cascade')
+    magento_sale_order_id = fields.Many2one(
+        'magento.sale.order',
+        'Magento Sale Order',
+        required=True,
+        ondelete='cascade',
+        select=True)
+    is_customer_notified = fields.Boolean(
+        'Customer notified',
+        help=MAGENTO_HELP)
+    is_visible_on_front = fields.Boolean(
+        'Visible on front',
+        help=MAGENTO_HELP)
+    status = fields.Char(
+        'Order status',
+        size=64,
+        help=MAGENTO_HELP)
+    backend_id = fields.Many2one(
+        'magento.backend', related='magento_sale_order_id.backend_id',
+        string='Magento Backend',
+        store=True,
+        readonly=True)
+    storeid = fields.Char(
+        'Store id',
+        help=MAGENTO_HELP)
+
+    @api.model
```

Comment:
I was sure it was inherited. This comment would tend to confirm that, but it could be wrong https://github.com/odoo/odoo/blob/9.0/openerp/api.py#L77
Had you ever had issues with that?